Earthfile

```diff
@@ -430,8 +419,8 @@ uki:
 # Copy uki artifacts into local build dir
 uki-local-artifacts:
     FROM +uki
-    COPY +version/VERSION ./
-    ARG VERSION=$(cat VERSION)
+    COPY +git-version/GIT_VERSION ./
```

Comment:
what is GIT_VERSION ? does that match the old VERSION?

Reply:
GIT_VERSION is just the version of kairos (with no k3s in it). It's just the result of `git describe --always --tags --dirty`.